test/e2e/aks.go

```diff
@@ -34,7 +34,28 @@ var _ = Describe("AKS cluster present", func() {
 		Expect(err).To(BeNil())
 	})
 
+	// TODO: remove this when all regions have the AKS
+	//       since this is going to happen in a weeks,
+	//       no need for external configuration option
+	regionsWithoutAKS := []string{
+		"australiacentral",
+		"australiacentral2",
+		"brazilsoutheast",
+		"eastus2euap",
+		"switzerlandwest",
+		"uaecentral",
+		"usgovvirginia",
+	}
+
 	It("should get kubeconfig", func() {
+		By("region = " + _env.Location())
+		for _, region := range regionsWithoutAKS {
+			// uses the region information stored in core environment, which reads it from instance metadata.
+			if _env.Location() == region {
```

Comment:
We need to use a `strings.EqualFold` check here instead of a direct comparison. Otherwise we will end up with [the same issue that was fixed in this PR](https://github.com/Azure/ARO-RP/pull/2521).